_docs_integrate/13-connector-error-codes.md

```diff
@@ -0,0 +1,79 @@
+| {% include custom-anchor name="error.runtime.relationshipTemplates.cannotCreateQRCodeForPeerTemplate" %}              | You cannot create a QRCode for a peer template.                                                                                                                           |
+| {% include custom-anchor name="error.runtime.relationshipTemplates.cannotCreateTokenForPeerTemplate" %}               | You cannot create a token for a peer template.                                                                                                                            |
+| {% include custom-anchor name="error.runtime.relationshipTemplates.invalidReference" %}                               | The reference for a relationship template must start with 'VE9L' or 'VE9L'. The given reference is not valid.                                                             |
+| {% include custom-anchor name="error.runtime.requestDeserialization" %}                                               | There was an error during the request deserialization.                                                                                                                    |
+| {% include custom-anchor name="error.runtime.servalError" %}                                                          | A serval error occurred.                                                                                                                                                  |
+| {% include custom-anchor name="error.runtime.startup.noActiveAccount" %}                                              | No AccountController could be found. You might have to login first.                                                                                                       |
+| {% include custom-anchor name="error.runtime.startup.noActiveConsumptionController" %}                                | No ConsumptionController could be found. You might have to login first.                                                                                                   |
+| {% include custom-anchor name="error.runtime.startup.noActiveExpander" %}                                             | No DataViewExpander could be found. You might have to login first.                                                                                                        |
+| {% include custom-anchor name="error.runtime.unknown" %}                                                              | An unknown error occurred. Check the error message or the stack trace to learn more.                                                                                      |
+| {% include custom-anchor name="error.runtime.unknownType" %}                                                          | Serval couldn't find an '@type' during a deserialization.                                                                                                                 |
```

Comment:
Niemand weiß, wer/was "Serval" ist. Es reicht wenn du schreibst, dass der type, der in `@type` angegeben ist, nicht existiert.